src/Mono.Android/Mono.Android.targets

```diff
@@ -174,23 +174,38 @@
     <ApiCompatibilityFiles Include="$(ApiCompatibilityDir)/reference/*.*" />
   </ItemGroup>
   <Target
-      Name="AfterBuild"
+      Name="_CheckApiCompatibility"
```

Comment:
are we not running apicompat check after build anymore?

Reply:
`AfterTargets` does the same thing while allowing the target to be explicitly executed for one-off test purposes.